### PR TITLE
[Merged by Bors] - Add demo nifi-kafka-druid-water-level-data

### PR DESCRIPTION
**Pull request by sbernauer**
## Description

To run
```
$ git checkout demo-nifi-kafka-druid-water-level-data
$ stackablectl --additional-demos-file demos/demos-v1.yaml demo install nifi-kafka-druid-water-level-data
```

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: demos/demos-v1.yaml [hunk 1]
```diff
@@ -45,6 +30,22 @@ demos:
...
+      - kafka
+      - druid
+      - superset
+      - minio
+      - s3
+      - water-levels
+    manifests:
+      - plainYaml: demos/nifi-kafka-druid-water-level-data/create-nifi-ingestion-job.yaml # TODO
+      - plainYaml: demos/nifi-kafka-druid-water-level-data/create-druid-ingestion-job.yaml # TODO
+      - plainYaml: demos/nifi-kafka-druid-water-level-data/setup-superset.yaml # TODO
```
Comment (sbernauer) [inline demos/demos-v1.yaml:48]:
```suggestion
      - plainYaml: https://raw.githubusercontent.com/stackabletech/stackablectl/main/demos/nifi-kafka-druid-water-level-data/create-nifi-ingestion-job.yaml # TODO
      - plainYaml: https://raw.githubusercontent.com/stackabletech/stackablectl/main/demos/nifi-kafka-druid-water-level-data/create-druid-ingestion-job.yaml # TODO
      - plainYaml: https://raw.githubusercontent.com/stackabletech/stackablectl/main/demos/nifi-kafka-druid-water-level-data/setup-superset.yaml # TODO
```

File: demos/nifi-kafka-druid-water-level-data/create-nifi-ingestion-job.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+kind: Job
+metadata:
+  name: create-nifi-ingestion-job
+spec:
+  template:
+    spec:
+      containers:
+        - name: create-nifi-ingestion-job
+          image: docker.stackable.tech/stackable/testing-tools:0.1.0-stackable0.1.0
+          command: ["bash", "-c", "curl -O https://raw.githubusercontent.com/stackabletech/stackablectl/demo-nifi-kafka-druid-water-level-data/demos/nifi-kafka-druid-water-level-data/IngestWaterLevelsToKafka.xml && python -u /tmp/script/script.py"]
```
Comment (sbernauer) [inline demos/nifi-kafka-druid-water-level-data/create-nifi-ingestion-job.yaml:12]:
```suggestion
          command: ["bash", "-c", "curl -O https://raw.githubusercontent.com/stackabletech/stackablectl/main/demos/nifi-kafka-druid-water-level-data/IngestWaterLevelsToKafka.xml && python -u /tmp/script/script.py"]
```

File: demos/nifi-kafka-druid-water-level-data/setup-superset.yaml [hunk 3]
```diff
@@ -9,17 +9,17 @@ spec:
       containers:
         - name: setup-superset
           image: docker.stackable.tech/stackable/testing-tools:0.1.0-stackable0.1.0
-          command: ["bash", "-c", "curl -o superset-assets.zip https://raw.githubusercontent.com/stackabletech/stackablectl/main/demos/kafka-druid-water-level-data/superset-assets.zip && python -u /tmp/script/script.py"]
+#          command: ["bash", "-c", "curl -o superset-assets.zip https://raw.githubusercontent.com/stackabletech/stackablectl/main/demos/nifi-kafka-druid-water-level-data/superset-assets.zip && python -u /tmp/script/script.py"]
+          command: ["bash", "-c", "curl -o superset-assets.zip https://raw.githubusercontent.com/stackabletech/stackablectl/demo-nifi-kafka-druid-water-level-data/demos/nifi-kafka-druid-water-level-data/superset-assets.zip && python -u /tmp/script/script.py"]
```
Comment (sbernauer) [inline demos/nifi-kafka-druid-water-level-data/setup-superset.yaml:13]:
```suggestion
          command: ["bash", "-c", "curl -o superset-assets.zip https://raw.githubusercontent.com/stackabletech/stackablectl/main/demos/nifi-kafka-druid-water-level-data/superset-assets.zip && python -u /tmp/script/script.py"]
```

**Review by maltesander — APPROVED** [review]
LGTM! Works. Only minor things.

File: docs/modules/ROOT/pages/demos/nifi-kafka-druid-water-level-data.adoc [hunk 4]
```diff
@@ -210,24 +213,111 @@ Topic measurements / Partition 0 / Offset: 7586541 / Timestamp: 1660831499070
...
+2. On the right it the other part iterating over all stations and emitting the current measurement in an endless loop
+
+You can zoom in by using your mouse and mouse wheel.
+
+image::demo-nifi-kafka-druid-water-level-data/nifi_3.png[]
+image::demo-nifi-kafka-druid-water-level-data/nifi_4.png[]
+
+The left workflows works as follows:
+
+1. The `Get station list` processors fetches the current list of stations as JSON via HTTP from the PEGELONLINE web service.
```
[the remark this answers is not in the document]
Comment (sbernauer) [reply docs/modules/ROOT/pages/demos/nifi-kafka-druid-water-level-data.adoc]: done

**Pull request closed without merging**